app/src/main/java/com/searonix/whentoplantapp/ui/home/HomeFragment.kt

```diff
@@ -1,19 +1,74 @@
 package com.searonix.whentoplantapp.ui.home
 
 import android.os.Bundle
+import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.TextView
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProviders
+import com.bumptech.glide.Glide
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.ktx.Firebase
+import com.google.firebase.storage.ktx.storage
 import com.searonix.whentoplantapp.R
+import kotlinx.android.synthetic.main.fragment_home.*
 
 class HomeFragment : Fragment() {
 
     private lateinit var homeViewModel: HomeViewModel
 
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+
+        val db: FirebaseFirestore = FirebaseFirestore.getInstance()
+
+        var imgview = image_view2
+        val storage = Firebase.storage
+
+        // Create a storage reference from app
+        val storageRef = storage.reference
+
+        //Create a reference with an initial file path and name
+        val pathReference = storageRef.child("apple-tree.jpg")
+        pathReference.downloadUrl.addOnSuccessListener {Uri->
+            val imageURL = Uri.toString()
+
+            Glide.with(this)
+                .load(imageURL)
+                .fitCenter()
+                .into(imgview)
+
+        }
+
+        db.collection("treeTypes")
+            .get()
+            .addOnCompleteListener { task ->
+                if (task.isSuccessful) {
+
+                    val treenames = task.result!!.map { snapshot ->
+                        snapshot["treeName"].toString()
+                    }
+
+                    val treeinfos = task.result!!.map { snapshot ->
+                        snapshot["info"].toString()
+                    }
+
+                    Log.d("fragmentLog", "list of treenames: + $treenames")
```

Comment:
define the tag globally as constant and use it everywhere instead typing it every time.